doc/nrf/known_issues.rst

```diff
@@ -2010,6 +2010,13 @@ NCSDK-12483: Missing debug symbols
 
   **Workaround:** Add the following text in the application :file:`CMakeLists.txt` file: ``zephyr_link_libraries(-Wl,--undefined=my_missing_debug_symbol)``.
 
+.. rst-class:: v1-9-0
+
+NCSDK-14015: Execution halts during boot.
+  When rpmsg, used by openthread and BT, is enabled on nrf53 simultaneously with TF-M the firmware does not boot.
```

Comment:
When the :kconfig:`CONFIG_RPMSG_SERVICE` is enabled on the nRF5340 DK together with TF-M, the firmware does not boot.
This option is used by OpenThread and Bluetooth modules.

Reply:
ok, but i'll replace DK with SoC.

Reply:
done